docs/getting_started.rst

```diff
@@ -289,258 +266,255 @@ Setting an attribute follows the ordinary convention for assigning attribute val
+list attribute of the parent component. The constructor for the ``SubComponent`` class 
+takes a ``Component`` as its only required argument. In this usage, the ``Component`` is "... analogous 
+to a blueprint or specification sheet for a biological part..." Whereas the ``SubComponent`` "... represents 
+the specific occurrence of a part..." within a larger design 
+(`SBOL version 3.0.0 specification document <https://sbolstandard.org/docs/SBOL3.0specification.pdf>`_). 
+For example, to add a promoter to a circuit design, first define the promoter and circuit as SBOL 
+``Component`` objects, then define a ``SubComponent`` as an instance of the promoter and add that 
+``SubComponent`` to the circuit's ``features`` attribute:
 
 .. code:: python
 
-    >>> point_mutation = SequenceAnnotation('PointMutation')
-    >>> target_promoter.sequenceAnnotations.add(point_mutation)
+    >>> ptet = sbol3.Component('pTetR', sbol3.SBO_DNA)
+    >>> ptet.roles = [sbol3.SO_PROMOTER]
+    
+    >>> circuit = sbol3.Component('circuit', sbol3.SBO_DNA)
+    >>> circuit.roles.append(sbol3.SO_ENGINEERED_REGION)
+    
+    >>> ptet_sc = sbol3.SubComponent(ptet)
+    >>> circuit.features += [ptet_sc]
 
 .. end
 
-Alternatively, the create method captures the construction and addition of the SequenceAnnotation in a single function call.  The create method ALWAYS takes one argument--the URI of the new object. All other values are initialized with default values. You can change these values after object creation, however.
-
-.. code:: python
-
-    >>> target_promoter.sequenceAnnotations.create('PointMutation')
-
-.. end
+-----------------------------------------
+Creating and Editing Reference Properties
+-----------------------------------------
 
-Conversely, to obtain a Python reference to the SequenceAnnotation from its identity:
+Some SBOL objects point to other objects by way of URI references. For example, Components point to their corresponding Sequences by way of a URI reference. These kind of properties correspond to white diamond arrows in UML diagrams, as shown in the figure above. Attributes of this type contain the URI of the related object.
 
 .. code:: python
 
-   >>> point_mutation = target_promoter.sequenceAnnotations.get('PointMutation')
+    >>> gfp = sbol3.Component('GFP', sbol3.SBO_DNA)
+    >>> gfp_seq = sbol3.Sequence('GFPSequence', elements='atgnnntaa', encoding=sbol3.IUPAC_DNA_ENCODING)
+    >>> gfp.sequences = [ gfp_seq ]
+    >>> print(gfp.sequences)
+    ['http://sbolstandard.org/testfiles/GFPSequence']
 
 .. end
 
-Or equivalently:
+Note that assigning the ``gfp_seq`` object to the ``gfp.sequences`` actually results in assignment of the object's URI. An equivalent assignment is as follows:
 
 .. code:: python
 
-   >>> point_mutation = target_promoter.sequenceAnnotations['PointMutation']
+    >>> gfp.sequences = [ gfp_seq.identity ]
+    >>> print(gfp.sequences)
+    ['http://sbolstandard.org/testfiles/GFPSequence']
```

Comment:
Add a example of using lookup() on an element of gfp.sequences

Reply:
Examples have been added.